scrapers/VirtualTaboo.yml

```diff
@@ -26,6 +26,9 @@ xPathScrapers:
       Performers:
         Name: $info//div[starts-with(@class,"info")]/a/text()
       Image: &imageSel //meta[@property="og:image"]/@content
+      Studio:
+        Name:
+          fixed: "Virtual Taboo"
```

Comment:
If we didn't want to us a fixed name `//meta[@name="author"]/@content` is available, but it doesn't have a space. I'm good with this path though.